<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Merchandise_Add</class>
 <widget class="QDialog" name="Merchandise_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Merchandise</string>
  </property>
  <widget class="QPushButton" name="AddMerchandise_BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddMerchandise_CommentsBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_NumberText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddMerchandise_CodeBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_KeyWordsText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Words</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_DateText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Date??</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="AddMerchandise_NumberBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddMerchandise_KeyValidation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddMerchandise_CodeValidation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CommentsText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="AddAlbum_DateBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddMerchandise_CommentsValidation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_TitleText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddMerchandise_TitleValidation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddMerchandise_CodeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddMerchandise_KeyWordsBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_CodeText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Authorisation Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddAlbum_MainText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add New Merchandise</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddMerchandise_TitleBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddMerchandise_BackButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Add</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddMerchandise_CodeButton</sender>
   <signal>clicked()</signal>
   <receiver>Merchandise_Add</receiver>
   <slot>AddMerchandise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>AddMerchandise()</slot>
 </slots>
</ui>
